docs/BR.md

```diff
@@ -1326,21 +1326,32 @@ No stipulation.
 
 The following SHALL apply for communicating the status of Certificates which include an Authority Information Access extension with an id-ad-ocsp accessMethod.
 
+Authoritative OCSP responses MUST be available (i.e. the responder MUST NOT respond with the "unknown" status) starting no more than 15 minutes after the Certificate's `notAfter` timestamp.
+
+The following SHALL apply for communicating the status of *all* Certificates for which an OCSP responder is willing to respond.
+
 OCSP responses MUST conform to RFC6960 and/or RFC5019. OCSP responses MUST either:
 
 1. Be signed by the CA that issued the Certificates whose revocation status is being checked, or
 2. Be signed by an OCSP Responder whose Certificate is signed by the CA that issued the Certificate whose
 revocation status is being checked.
 
-In the latter case, the OCSP signing Certificate MUST contain an extension of type `id-pkix-ocsp-nocheck`, as
-defined by RFC6960.
+In the latter case, the OCSP signing Certificate MUST contain an extension of type `id-pkix-ocsp-nocheck`, as defined by RFC6960.
```

Comment:
Consider requiring the delegated OCSP responder cert to comply with the OCSP responder cert profile in section 7.

This may require that some OCSP responder certifcates be replaced if issued prior to 2023-09-15, but that's probably not the worst thing since a OCSP responder key shouldn't be used for more than a year anyway.

Reply:
Yeah, that's a good idea. This PR hasn't actually changed this line yet (it only appears here due to a line-wrapping thing) but I think that makes sense to fit with this same ballot.